reactor-core/src/test/java/reactor/core/publisher/DirectProcessorTest.java

```diff
@@ -133,10 +133,10 @@ public void error() {
 
         tp.subscribe(ts);
 
-        Assert.assertTrue("No subscribers?", Scannable.from(tp).inners().count() != 0);
-        Assert.assertFalse("Completed?", tp.hasCompleted());
-        Assert.assertNull("Has error?", tp.getError());
-        Assert.assertFalse("Has error?", tp.hasError());
+	    assertThat(tp.hasSubscriber()).as("hasSubscriber").isTrue();
```

Comment:
note for reviewers: `DirectProcessorTest` should ideally be re-formatted to use tabs consistently. in order not to pollute this PR's commit, I've only let the IDE reintroduce tabs on modified lines.